builtin/game/item.lua

```diff
@@ -309,7 +309,7 @@ function core.item_place_node(itemstack, placer, pointed_thing, param2)
 
 	-- Run script hook
 	local _, callback
-	for _, callback in ipairs(core.registered_on_placenodes) do
+	for _, callback in pairs(core.registered_on_placenodes) do
```

Comment:
`pairs` is slightly faster than `ipairs`. The order doesn't matter here, so this change LGTM.